file-tracker/file_tracker/client_connection.py

```diff
@@ -0,0 +1,51 @@
+import json
+import logging
+
+from aiortc import RTCPeerConnection, RTCSessionDescription
+from file_operations import ls, tail
+
+# STUN/TURN server configuration
+ICE_SERVERS = [{
+    "urls": ["stun:34.79.246.4:3478"]
+}, {
+    "urls": ["turn:34.79.246.4:3478"]
```

Comment:
can we launch https://github.com/coturn/coturn on a docker container with GCP cloud run?